client/src/components/Searchbar.vue

```diff
@@ -134,14 +134,13 @@ export default defineComponent({
         initializeSearch(): void {
             if (this.currentKeyword.trim().length > 0) {
                 this.addKeyword();
+                this.$emit('onSearch', { searchTerms: this.searchTerms as string[] | [] });
             }
 
             // if no search terms are specified we need to prevent the state change.
             if (!this.searchTerms || this.searchTerms.length === 0 || this.$route.path === '/search') {
                 return;
             }
-
-            this.$emit('onSearch', { searchTerms: this.searchTerms as string[] | [] });
```

Comment:
This change is not correct since now you are only triggering the search if a user has typed a keyword which is not yet converted into a chip

Reply:
yeah but if we first want to convert it into chip then we don't need this ticket or it should be converted and trigger the search in the same time?

Reply:
FWIW, this works as expected for me. I can enter in a term, press enter and it searches.

Reply:
This thread is based on an older version of the file.